src/pivot.jl

```diff
@@ -35,10 +35,10 @@ function pivot(A::Matrix{Float64},
     P = BitArray(q)
 
     x[P] =  A[:,P] \ b
-    y[~P] =  A[:,~P]' * (A[:,P]*x[P] - b)
+    y[@__dot__(!P)] =  A[:,@__dot__(!P)]' * (A[:,P]*x[P] - b)
```

Comment:
I really don't love having `@__dot__` everywhere, but I'm not sure there's a better way. `~P` is deprecated in v0.6, and `.~(P)` is a syntax error in v0.5. 

One alternative would be to define:

```julia
const not = !
```

and then do `not.(P)` instead of `~P`. That's essentially what the `@__dot__` macro is doing.